### ISSUE TITLE
PomScmPluginLauncherSpec fails when plugin was cloned using https

### ISSUE THREAD

**Issue by szpak** [nebula-plugins/nebula-publishing-plugin#27]
`PomScmPluginLauncherSpec` fails when `nebula-publishing-plugin` was cloned using `https://` Git address (not `git://`).

```
nebula.plugin.publishing.maven.PomScmPluginLauncherSpec > look in pom FAILED
    Condition not satisfied:

    pom.scm.url.text() == 'scm:git@github.com:nebula-plugins/nebula-publishing-plugin.git'
    |   |   |   |      |
    |   |   |   |      false
    |   |   |   |      8 differences (87% similarity)
    |   |   |   |      scm:(ht)t(ps://)github.com(/)nebula-plugins/nebula-publishing-plugin.git
    |   |   |   |      scm:(gi)t(@----)github.com(:)nebula-plugins/nebula-publishing-plugin.git
    |   |   |   scm:https://github.com/nebula-plugins/nebula-publishing-plugin.git
    |   |   scm:https://github.com/nebula-plugins/nebula-publishing-plugin.git
    |   scm:https://github.com/nebula-plugins/nebula-publishing-plugin.gitscm:https://github.com/nebula-plugins/nebula-publishing-plugin.git
    4.0.0look-in-pomunspecifiedpom1.0#look-in-pom;unspecifiedunspecifiedreleasefooLinux2015-02-01_22:22:502.2.1/build/test/nebula.plugin.publishing.maven.PomScmPluginLauncherSpec/look-in-pomhttps://github.com/nebula-plugins/nebula-publishing-plugin.git96713e8localhost.localdomainLOCALLOCALLOCAL1.8.0_31-b13 (Oracle Corporation)1.8.0_31scm:https://github.com/nebula-plugins/nebula-publishing-plugin.gitscm:https://github.com/nebula-plugins/nebula-publishing-plugin.githttps://github.com/nebula-plugins/nebula-publishing-plugin
        at nebula.plugin.publishing.maven.PomScmPluginLauncherSpec.look in pom(PomScmPluginLauncherSpec.groovy:42)
```

**Comment by bmuschko** [comment]
Good find. Would you be interested in providing a fix that dynamically determines the SCM URL?

**Comment by nadavc** [comment]
Fixed in 797f0965073b773d48ab5dd4c83452b357c0a098
